src/methods/function.js

```diff
@@ -600,16 +601,19 @@ function InternalCall(realm: Realm, F: FunctionValue, thisArgument: Value, argsL
     invariant(realm.getRunningContext() === callerContext);
 
     for (let t2 of realm.tracers)
-      t2.afterCall(F, thisArgument, argsList, undefined, result);
+      t2.afterCall(F, thisArgument, argsList, undefined, (result: any));
```

Comment:
@NTillmann I am not sure improvements to Flow, which we discussed recently, include reasoning about control-flow infeasibility.